src/ReasonPhrase.php

```diff
@@ -104,110 +104,104 @@
 
     public static function fromName(string $name): ReasonPhrase
     {
-        $reasonPhrase = ReasonPhrase::tryFromName($name);
-        if (is_null($reasonPhrase)) {
-            $enumName = static::class;
-            throw new ValueError("$name is not a valid name for enum \"$enumName\"");
+        $reasonPhrase = self::tryFromName($name);
+
+        if (null === $reasonPhrase) {
+            throw new ValueError(sprintf('%s is not a valid name for enum "%s"', $name, static::class));
```

Comment:
Why change all interpolated strings to `sprintf`? It's longer and harder to read IMO, even if you do save one line.

Reply:
This is an habit of mine, when I have to "build" strings, I use `sprintf`. I don't like using string concatenation in PHP, I found it very messy.

Reply:
> This is an habit of mine, when I have to "build" strings, I use `sprintf`. I don't like using string concatenation in PHP, I found it very messy.

Understandable, though I find it the other way round. Well, it doesn't matter much for short strings with small amount of parameters really.

Reply:
This repo uses interpolated strings, as those generally have the lowest cognitive load of the possible options, and are also easy to grasp for newcomers, and hard to fuck up if you have at least a semi-decent IDE which gives those variables inside the string a different color.

Reply:
Do you want me to revert the changes?

Reply:
Yeah, let's not add unneeded changes just for the sake of it.

Reply:
Right, I'm away from the computer now, I will do it today but just a bit later.

Reply:
Ok, got delayed here, so I did it. Let me know if it's ok now.